docs/examples/gen_velocities.rst

```diff
@@ -0,0 +1,432 @@
+*********************************************
+Generating synthetic scintillation velocities
+*********************************************
+
+This tutorial describes how to generate synthetic scintillation velocities for
+a pulsar in a circular orbit and a single one-dimensional screen, both with
+known parameters.
+
+Further explanations and derivations of the equations seen here can be found in
+Daniel Baker's "`Orbital Parameters and Distances
+<https://eor.cita.utoronto.ca/images/4/44/DB_Orbital_Parameters.pdf>`_"
```

Comment:
I believe this is currently only accessible for people who have access to the wiki.

Reply:
I'm not sure, but I doubt that Daniel can change the permissions. I think he'd have to host the file outside the wiki.

It would indeed be nice to include the derivation here (later).

We could also add a link here to [your derivations](http://www.astro.utoronto.ca/~mhvk/scintillometry.html#org5ea6450)

Reply:
Yes, fine to add a link to my write-up.

Reply:
Done.